File: Scala_Practice/ScalaExamples/src/main/java/com/java/examples/session_2/Car.java
package com.java.examples.session_2;

public class Car {
	private final int year;
	private int milesDriven;

	public Car(int yearOfMake) {
		year = yearOfMake;
	}

	public int getYear() {
		return year;
	}

	public int getMileDriven() {
		return milesDriven;
	}

	public void drive(int distance) {
		milesDriven += Math.abs(distance);
	}
	
	
	public static void main(String[] args) {
		Car c = new Car(2014);
		c.drive(100);
		System.out.println(c.getMileDriven());
	}
}

File: Scala_Practice/ScalaExamples/src/main/scala/com/scala/examples/session2/Car.scala
package com.scala.examples.session2


class Car(val year: Int) {
  
  private var milesDriven: Int = 0
  def miles() = milesDriven
  def drive(distance: Int) = milesDriven += Math.abs(distance)
  
  
  
}


object Car{
  
   def main(args: Array[String]): Unit = {
    val car = new Car(220)

    car.drive(200)

    println(car miles)
    
    
  }
  
}

